bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -58,7 +58,8 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
     {%+ else -%}
         {{ task.task_name }} = bigquery_etl_query(
             task_id='{{ task.task_name }}',
-            destination_table={%+ if task.destination_table -%}'{{ task.destination_table }}'
+            destination_table={%+ if task.date_partition_offset -%}'{{ task.destination_table }}${% raw %}{{{% endraw %} macros.ds_format(macros.ds_add(ds, {{ task.date_partition_offset }}), "%Y-%m-%d", "%Y%m%d") {% raw %}}}{% endraw %}'
```

Comment:
Preserving relud's TODO from the `metadata.yaml` files:
```suggestion
            {#+ TODO when Airflow is updated to 2.2+ use ds_nodash instead of ds_format -#}
            destination_table={%+ if task.date_partition_offset -%}'{{ task.destination_table }}${% raw %}{{{% endraw %} macros.ds_format(macros.ds_add(ds, {{ task.date_partition_offset }}), "%Y-%m-%d", "%Y%m%d") {% raw %}}}{% endraw %}'
```